tensorflow_quantum/python/differentiators/gradient_test.py

```diff
@@ -27,21 +27,37 @@
 from tensorflow_quantum.python.differentiators import parameter_shift
 from tensorflow_quantum.core.ops import circuit_execution_ops, batch_util
 
-DIFFS = [
+ANALYTIC_DIFFS = [
     linear_combination.ForwardDifference(grid_spacing=0.0001),
     linear_combination.ForwardDifference(error_order=2, grid_spacing=0.0001),
     linear_combination.CentralDifference(grid_spacing=0.0001),
     linear_combination.CentralDifference(error_order=4, grid_spacing=0.0001),
     parameter_shift.ParameterShift(),
 ]
 
-OPS = [
+SAMPLED_DIFFS = [
+    linear_combination.ForwardDifference(grid_spacing=0.1),
+    linear_combination.CentralDifference(grid_spacing=0.1),
+    parameter_shift.ParameterShift(),
+]
+
+SAMPLED_DIFFS_TOLS = [0.5, 0.5, 0.2]
```

Comment:
Here also the tolerances seem a bit loose, could they be made tighter to like `1/10` as large?